### PR TITLE
Template tweaks and drop Internet Explorer support

### PR DESCRIPTION
**Pull request by XhmikosR**
* switch to plain `doctype`
* remove unneeded type=text/css from style attributes
* beautify CSS
* remove svg4everybody

I only checked the CSS mode, need more eyes for the rest :)

### CODE REVIEW

File: test/tmpl/css.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+<!doctype html>
+<html lang="en">
     <head>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-        <title>svg-sprite preview</title>
-        <link href="{{{css}}}" rel="stylesheet" type="text/css"/>
+        <meta charset="utf-8">
+        <title>SVG CSS sprite preview | svg-sprite</title>
+        <link href="{{{css}}}" rel="stylesheet">
         <style>
```
[the remark this answers is not in the document]
Comment (Kreeg) [reply test/tmpl/css.html:7]: @XhmikosR that was not me :D In `tmpl/symbol.html` I copy-pasted them from this file. I'm sure we can move them to css file.

[the remark this answers is not in the document]
Comment (XhmikosR) [reply test/tmpl/css.html:7]: Oh, sorry, I thought you did it :)

No worries then, we can improve these later.

File: tmpl/css/sprite.html [hunk 2]
```diff
@@ -1,21 +1,117 @@
...
+                margin: 0;
+                font-size: .7em;
+            }
+
+            footer a {
+                color: #0f7595;
+                margin-left: 0;
+            }
+        </style>
+        <style>
```
Comment (XhmikosR) [inline tmpl/css/sprite.html:98]: I will combine this with the above and just add a comment

**Comment by XhmikosR** [comment]
@Kreeg how about we move with this before #613 ?

File: test/tmpl/symbol.html [hunk 3]
```diff
@@ -44,10 +44,6 @@
             figcaption {
                 display: none;
             }
-            i {
```
Comment (XhmikosR) [inline test/tmpl/symbol.html:47]: Removed the selector from test files that don't have `i` elements.

**Review by Kreeg — APPROVED** [review]
Looks good

**Pull request merged**